src/test_util.rs

```diff
@@ -1,5 +1,13 @@
 use Uuid;
 
+cfg_if! {
```

Comment:
wait on one of my PRs that is adding a public accessible constants for this

Reply:
wait for #212

Reply:
@kinggoesgaming #212 is targetted to 0.7.0 and this is for 0.6.4

Reply:
Let's see if I can retarget the branch